test/WebSites/ActionConstraintsWebSite/Startup.cs

```diff
@@ -9,18 +9,19 @@ namespace ActionConstraintsWebSite
 {
     public class Startup
     {
-        public void Configure(IApplicationBuilder app)
+        // Set up application services
+        public void ConfigureServices(IServiceCollection services)
         {
-            var configuration = app.GetTestConfiguration();
-
-            app.UseServices(services =>
+            services.AddMvc();
+            services.ConfigureMvc(options =>
             {
-                services.AddMvc();
-                services.Configure<MvcOptions>(options =>
-                {
-                    options.AddXmlDataContractSerializerFormatter();
-                });
+                options.AddXmlDataContractSerializerFormatter();
             });
+        }
+
+        public void Configure(IApplicationBuilder app)
+        {
+            var configuration = app.GetTestConfiguration();
```

Comment:
My point is that this is an issue all over the test code base, I don't see the value in fixing one place, all that's going to do is make any future FindReplace All operations harder.

If this is something you think is worth fixing, I think it should be done as one PR doing a global replace all removing the var from all tests at once.